android/src/toga_android/widgets/label.py

```diff
@@ -66,9 +66,12 @@ def set_alignment(self, value):
         if not self.native.getLayoutParams():
             return
 
-        # Justified text wasn't added until Android O (SDK 26)
-        if value == JUSTIFY and Build.VERSION.SDK_INT >= Build.VERSION_CODES.O:
-            self.native.setJustificationMode(LineBreaker.JUSTIFICATION_MODE_INTER_WORD)
-        else:
-            self.native.setJustificationMode(LineBreaker.JUSTIFICATION_MODE_NONE)
+        # Justified text wasn't added until API level 26.
+        if Build.VERSION.SDK_INT >= 26:
```

Comment:
I prefer to use a plain number rather than a VERSION_CODES constant, because that's how the API documentation is written (e.g. "Added in API level 29").